ebean-core/src/main/java/io/ebeaninternal/server/deploy/parse/AnnotationParser.java

```diff
@@ -129,4 +129,11 @@ protected void setColumnName(DeployBeanProperty prop, String name) {
     }
     return columnNames;
   }
+
+  /**
+   * Process any formula from &#64;Formula or &#64;Where.
+   */
+  protected String processFormula(String source) {
```

Comment:
Every argument passed by `@Where` / `@Formula` is currently routed over this method.
My future plans are, to implement a "DeployParser" alternative, so that you can register your own functions somewhere in the serverConfig: If you write something like  `ebean_BitAnd(field1, field2)` this can invoke the registered method and return a platform dependent result.